### ISSUE TITLE
Remove github repo info from PRO Foundry page

### ISSUE THREAD

**Issue by nataled** [OBOFoundry/OBOFoundry.github.io#1794]
PRO cannot use GitHub for repo due to its size, so all of the information (stars, contributors, last commit) is meaningless and, actually, very misleading. If there is a way to do it, please remove the github banner from https://obofoundry.org/ontology/pr.html

**Comment by matentzn** [comment]
I agree, there should be an option to hide GitHub stars on the site.
